Source/WebCore/dom/Document.h

```diff
@@ -2343,6 +2352,9 @@ class Document
     unsigned m_dataListElementCount { 0 };
 
     OptionSet<ListenerType> m_listenerTypes;
+    MemoryCompactRobinHoodHashMap<AtomString, unsigned> m_eventListenerCounts;
```

Comment:
How many enum do we have? How about having EnumeratedArray instead?

Reply:
As far as I can tell, using EnumeratedArray instead of MemoryCompactRobinHoodHashMap doesn't yield any additional benefit.